apps/docs/content/components/navigation/Disclosure.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Disclosure
+description: The disclosure component is used to put long sections of information under a block that users can expand or collapse.
+category: "navigation"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/Disclosure/src/Disclosure.tsx
+    aria: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/
+---
+
+<Example src="Disclosure/docs/preview" isOpen />
```

Comment:
We should add a section in this page that talks about the progressive enhancement, and how it's normal that there is no animation in certain non-modern browsers